ts/screens/profile/ServicesPreferenceScreen.tsx

```diff
@@ -82,6 +82,7 @@ const ServicesPreferenceScreen = (): ReactElement => {
       pot.isSome(profile) &&
       prevMode !== profileServicePreferenceMode
     ) {
+      IOToast.hideAll();
```

Comment:
As above, `IOToast` ref can be replaced with `useIOToast` hook, what do you think?

Reply:
yes, good suggestion